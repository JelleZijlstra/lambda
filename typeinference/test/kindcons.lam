data option a = None | Some (option a int) in
42
