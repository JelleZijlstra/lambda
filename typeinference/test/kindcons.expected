%%stdout%%
Error: Cannot unify kinds: * and (* -> kindvar1)
%%stderr%%
